pages/desktop.js

```diff
@@ -45,6 +45,24 @@ class Atom extends React.Component<OSProps> {
           </Hero.Pane>
         </Hero>
         <ContentSections>
+          <ContentSection>
+            <ContentSection.Pane>
+              <ContentSection.Title>What is nteract?</ContentSection.Title>
+              <Type.p>
+                nteract is a next-gen React-based UI for Jupyter notebooks. It
+                provides a simple, intuitive interface and offers several
+                improvements over the classic Jupyter UI, such as inline cell
+                toolbars, drag and droppable cells, and a built-in data
+                explorer.
+              </Type.p>
+            </ContentSection.Pane>
+            <ContentSection.Pane visual>
+              <Image>
+                <src="/static/notebook-vega-altair.png"
```

Comment:
There's an extra `<` at the start here that needs to be deleted to fix the build here.

```suggestion
                src="/static/notebook-vega-altair.png"
```